### PR TITLE
Cypress testing for Key Insights, other optimizations

### PR DESCRIPTION
**Pull request by henrygoodman**
- Added Cypress test for Key insights
- Improved test time by removing delay on tests with `cy.type()`
- Moved Cypress testfile to dedicated folder

### CODE REVIEW

**Review by gracebilliris — APPROVED** [review]
Looks good!

**Review by srujankarthik — APPROVED** [review]
looks good!

File: cypress/e2e/5SummaryFeature.cy.js [hunk 1]
```diff
@@ -12,15 +12,14 @@ describe("Home", () => {
...
+            { delay: 0 }
+          );
 
         cy.get("#summarise-button-id").click();
 
-        cy.wait(12000);
-
         // Assert that the text contains the sentiment and score
-        cy.get("#summary-result").then(($div) => {
+        cy.get("#summary-result", {timeout: 20000}).then(($div) => {
```
Comment (stephenb-coding) [inline cypress/e2e/5SummaryFeature.cy.js]: btw `.then()` will not automatically retry the command until the assertion passes. `.should()` is more robust and will retry the assertion until it passes or the timeout finishes

**Review by stephenb-coding — APPROVED** [review]

File: cypress/e2e/5SummaryFeature.cy.js (hunk 1 above)
Comment (henrygoodman) [reply cypress/e2e/5SummaryFeature.cy.js]: Good pickup, have updated this test to now use `.should()`

**Pull request merged**